lib/manager/gitlabci/extract.ts

```diff
@@ -7,12 +7,17 @@ import type { ExtractConfig, PackageDependency, PackageFile } from '../types';
 import type { GitlabPipeline } from './types';
 import { replaceReferenceTags } from './utils';
 
+const commentsRe = /^\s*#/;
+const whitespaceRe = /^(?<whitespace>\s*)/;
+const imageRe = /^(?<whitespace>\s*)image:\s*'?"?(?<image>[^\s'"]+|)'?"?\s*$/;
+const nameRe = /^\s*name:\s*'?"?(?<depName>[^\s'"]+|)'?"?\s*$/;
+const serviceRe = /^\s*-\s*(?:name:\s*)?'?"?(?<depName>[^\s'"]+)'?"?\s*$/;
```

Comment:
```suggestion
const imageRe = /^(?<whitespace>\s*)image:\s+'?"?(?<image>[^\s'"]+|)'?"?\s*$/;
const nameRe = /^\s*name:\s+'?"?(?<depName>[^\s'"]+|)'?"?\s*$/;
const serviceRe = /^\s*-\s*(?:name:\s+)?'?"?(?<depName>[^\s'"]+)'?"?\s*$/;
```

I think at least one whitespace is required after a yaml key